cli/build.gradle

```diff
@@ -180,4 +180,18 @@ test {
 		dependsOn project.jar
 		systemProperty 'sampleTemplateJarFile', project.jar.archivePath
 	}
+
+	def parallelForks = 1
+	def cores = Runtime.runtime.availableProcessors()
+
+	if (cores > 4) {
+
+		parallelForks = cores - 2
+	}
+	else if (cores > 2) {
+
+		parallelForks = cores - 1
+	}
+
+	maxParallelForks = parallelForks
```

Comment:
By making tests parallel, this breaks debugging (your debugger stops on first parallel tests but others keep going and new jvms are forked for each new test).  So update this code so that when debugger is enabled `--debug-jvm` is passed, then we don't run tests in parallel.